app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -11,29 +11,36 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/pinPasswordBackground">
```

Comment:
This color is also different, check mocks. Replace this with `oppiaLightGreen`

Reply:
Done.